eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -14,5 +14,13 @@
     <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.CSharp.CodeStyle/*" />
     <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.VisualBasic.CodeStyle/*" />
     <UsagePattern IdentityGlob="Microsoft.Net.Compilers.Toolset/*" />
+
+    <!-- Roslyn's source-build CI builds both NetPrevious and NetCurrent. This 7.0 ref pack shows up as
+         prebuilt only for the repo CI build but not full source-build. -->
+    <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Ref/7.0*" />
```

Comment:
Just curious, where does roslyn use the aspnetcore targeting pack?

Reply:
It stems from this - https://github.com/dotnet/roslyn/blob/d080175cabfe297ebf079af099279b61913bcc28/eng/targets/Settings.props#L79

Reply:
Looks like maybe some stuff for Razor? src/Tools/ExternalAccess/AspNetCore/Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.csproj